controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -321,21 +304,18 @@ public void refresh() {
     }
 
     @Override
-    CompletableFuture<Void> createHistoryTable(Data<Integer> data) {
+    CompletableFuture<Void> createHistoryTableIfNotExists(Data<Integer> data) {
```

Comment:
There is probably a bit of taste involved in this so I don't want to press too much, but just as a way of example, a create call in zookeeper only creates a znode in the case it doesn't exist. We decided to not call it `createIfNotExists` as the suffix seems unnecessary and can be clarified by presenting the semantics clearly.